packages/rest/src/rest.server.ts

```diff
@@ -336,11 +334,19 @@ export class RestServer extends Context implements Server, HttpServerLike {
   private async _serveOpenApiSpec(
     request: Request,
     response: Response,
-    options?: OpenApiSpecOptions,
+    settings?: OpenApiSpecOptions,
```

Comment:
This looks suspicious to me - a `settings` argument having type `OpenApiSpecOptions`? Let's revert this change or use a more descriptive name, e.g. `specOptions`.